package.json

```diff
@@ -16,8 +16,8 @@
   "scripts": {
     "test": "jest",
     "storybook": "run-s build:svgs:index storybook:dev",
-    "storybook:dev": "start-storybook -p 9001 -c .storybook -s ./public",
-    "build-storybook": "build-storybook -c .storybook -o .out -s ./public",
+    "storybook:dev": "start-storybook --port 9001 --config-dir .storybook --static-dir ./public",
```

Comment:
I had mistakenly thought `-s` might've been related to a change in `https` functionality, so I changed this to the verbose versions to reduce confusion.